docs/sources/setup-grafana/configure-security/configure-authentication/okta/index.md

```diff
@@ -8,61 +8,100 @@ weight: 1200
 
-1. Pick **Web** as the platform.
+1. In the **Application type** field, select `Web Application` and then click **Next**.
 
-1. Enter a name for your application (or leave the default value).
+1. Complete the following **Web App Integration Options** fields:
 
-1. Add the **Base URI** of your application, such as https://grafana.example.com.
+   | Field                             | Description                                                                                                                                                 |
+   | --------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | App integration name              | Enter an application name.                                                                                                                                  |
+   | Logo (Optional)                   | Add a logo.                                                                                                                                                 |
+   | Grant type                        | Select `Authorization Code`.                                                                                                                                |
+   | Sign-in redirect URIs             | If you have a Grafana Cloud account, enter `https://<org>.grafana.net/login/okta`. If you self-host Grafana, enter `https://<base grafana url>/login/okta`. |
+   | Sign-out redirect URIs (Optional) | If you have a Grafana Cloud account, enter `https://<org>.grafana.net/login/okta`. If you self-host Grafana, enter `https://<base grafana url>/login/okta`. |
+   | Base URIs (Optional)              | If you have a Grafana Cloud account, enter `https://<org>.grafana.net`. If you self-host Grafana, enter `https://<base grafana url>`.                       |
+   | Controlled access                 | Select whether to assign the app integration to everyone in your org, only selected group(s), or skip the assignment until after app creation.              |
 
-1. Enter values for the **Login redirect URI**. Use **Base URI** and append it with `/login/okta`, for example: https://grafana.example.com/login/okta.
+1. Click **Done**.
 
-1. Click **Done** to finish creating the Okta application.
+## Enable Okta OAuth in Grafana Instance
 
-## Enable Okta OAuth in Grafana
+This section includes tasks you must complete to enable Okta OAuth in self-hosted Grafana or Grafana Cloud.
 
-1. Add the following to the [Grafana configuration file]({{< relref "../../../configure-grafana/#config-file-locations" >}}):
+### Enable Okta OAuth in self-hosted Grafana
+
+If you are hosting your own Grafana instance, ensure it is [accessed over HTTPS]({{< relref "../../../set-up-https" >}}). Okta requires secure communcation for exchanging authentication information between the browser and the idenitity provider.
```

Comment:
@Mkrummz , The HTTPS docs are a brand-new addition to the docs! Can you please verify that these are the docs we want to send users to?